cicd/forgeops-tests/lib/utils/kubectl.py

```diff
@@ -16,50 +15,80 @@
 KUBECTL_COMMAND = 'kubectl'
 
 
-def exec(namespace, command):
+def exec(namespace, sub_command):
     """
     Run a kubectl exec command
-    :param command: command to run
+    :param namespace: kubernetes namespace
+    :param sub_command: command to run
     :return: (stdout, stderr) from running the command
     """
-    command = ' '.join([KUBECTL_COMMAND, '-n', namespace, 'exec', command])
+
+    assert isinstance(namespace, str), 'Invalid namespace type %s' % type(namespace)
```

Comment:
No really fan of having every time asserts at the beginning of each methods. 
Looks like more unit tests to me, we can add unit tests on our libs is needed.

Reply:
Okay, yes the are a bit messy.  However, I do think they offer a clear contract of what is expected from the calling code.  It is more of an aid to the test developer than a unit test.  I am happy to remove them and rely on Python's "duck typing"

I think adding unit tests to our libs is a great idea.